SnakeAndLadder/src/PlayerClass.java

```diff
@@ -0,0 +1,26 @@
+
+public class PlayerClass {
+	private  int currentposition=0;
+	public int getCurrentPosition()
+	{
+		return currentposition;
+	}
+	public void isLadder(int number)

```

Comment:
How can a player be ladder or snake or normal?

Reply:
either this class does not denote a Player or the methods you created here should not be here